Sources/CodeEditTextView/TextLayoutManager/TextLayoutManager.swift

```diff
@@ -232,8 +234,8 @@ public class TextLayoutManager: NSObject {
     func layoutLines() { // swiftlint:disable:this function_body_length
         guard let visibleRect = delegate?.visibleRect, !isInTransaction, let textStorage else { return }
         CATransaction.begin()
-        let minY = max(visibleRect.minY, 0)
-        let maxY = max(visibleRect.maxY, 0)
+        let minY = max(visibleRect.minY - 350, 0) // +-350px for a bit padding while laying out lines.
```

Comment:
This could be seen as a magic number. Should this be configurable?

Reply:
I hadn't considered that. It is a fairly magic number and I know NSTableView does a similar un-configurable magic. I can add it as a public variable on `TextLayoutManager` in case it needs configuring.